### ISSUE TITLE
I'm at peace now

### ISSUE THREAD

**Issue by mukarramkhalid** [deanoemcke/thegreatsuspender#1329]
I'd like to express my gratitude to the new maintainer who messed up this project.

When Google chrome removed this extension automatically and I lost about `900` tabs across different browser profiles, I just looked upon my life and realized that I had a severe mental illness and a problem. I would leave the tabs open and never go back to them. I always convinced myself that one day I would need that tab again. But guess what, I never did.

Because of this new self-realization, I'm not even going to try to bring back the tabs I lost. 

I have finally achieved inner peace.

**Comment by yehuizhang** [comment]
S.I.P!

**Comment by ITLINKSLTD** [comment]
@mukarramkhalid
I can save your life maybe
Go to Chrome history and type : klbibkeccnjlkjkiokjodocebajanakg

**Comment by mukarramkhalid** [comment]
Don't tempt me. I'm on a relapse.

**Comment by ITLINKSLTD** [comment]
@mukarramkhalid I am serious! try it and you will find all your lost tabs

**Comment by attemarkula** [comment]
That's no mental illness, you are just classical archivist. I still can't drop my "threads".

But, as I see that you are a bit hard on your self about the loss of something, never let a good crisis go to waste.

Chose now, a single thing on your life that you see is wrong and make it better. This worked for me 3 times. And it will work only if that crisis reminds you to stay true.

**Comment by justingoldberg** [comment]
I am like this too. Thankfully freshstart and sessionbuddy can tabs to a file.

There's this [faq ](https://github.com/greatsuspender/thegreatsuspender/issues/526) but basically search for klbibkeccnjlkjkiokjodocebajanakg and open them in new tabs, then remove all of it except for the part after _uri=_


Oddly, I just discovered Dean's discarder extension, which uses the official tab discard api, and it's still working:
https://chrome.google.com/webstore/detail/the-great-discarder/jlipbpadkjcklpeiajndiijbeieicbdh

**Comment by munael** [comment]
Doesn't TGS support such discarding as well?

![image](https://user-images.githubusercontent.com/5769148/108042537-997d1600-7048-11eb-818f-4082f11cb3c7.png)

I'm using a fork off of https://github.com/aciidic/thegreatsuspender-notrack now. Should switch to https://github.com/gioxx/MarvellousSuspender though, probably.

**Comment by electrikmilk** [comment]
It's alright man, everyone's lost their pr0n tabs every now and again, glad you were able to come to terms with it.

**Comment by schittli** [comment]
Is there any good reason to keep such 'issues' open and not close them?

As a developer, I would have a crisis if I see 360 open issues.

And I would have an even bigger crisis if I had to read every such useless issue a second time to see if I have missed important information...

**Comment by mukarramkhalid** [comment]
@schittli 

Looks like you're too late to the party .. 

So let me TLDR for you only because you're too stupid to read the room and do any research on your own.

### TLDR

- The author handed over this project to someone else.
- The new person embedded malware in the latest release.
- Google banned the project and removed it from Chrome Web Store.
- People confronted the new Author.
- He disappeared .. never coming back ..
- fu*k you for grave digging an old issue just to be an as*hole.

Best,

**Comment by attemarkula** [comment]
No, no, tell him how you really feel. Make him remember forever. 

Yeah f this thred.

**Comment by mukarramkhalid** [comment]
Oh look another fuktard @attemarkula outta nowhere, spewing shit, trying to be noticeable.

**Comment by attemarkula** [comment]
I was already out and they pull me back in... Why you be drama queen again?
Just fix shit and be happy.

**Comment by mukarramkhalid** [comment]
Great. Now you're making no sense. Be a doll and fuk off. We have noticed you already ffs. Stop trying so hard.
